TypeCobol/Compiler/Scanner/MultilineScanState.SpecialNamesContext.cs

```diff
@@ -19,20 +19,24 @@ public class SpecialNamesContext : IEquatable<SpecialNamesContext>
             /// NB : value will be null until at least one symbolic character is defined
             /// => only use method AddSymbolicCharacter to safely add an element to this list
             /// </summary>
-            public IList<string>? SymbolicCharacters { get; private set; }
+            private HashSet<string>? _symbolicCharacters;
```

Comment:
We could add the check for duplicates later but this would not change the meaning of the declarations: the `SpecialNamesContext` should still hold unique values. This also allows for case insensitive comparisons.